Sources/Sentry/include/HybridPublic/SentryEnvelope.h

```diff
@@ -63,6 +63,8 @@ SENTRY_NO_INIT
 
 @property (nullable, nonatomic, readonly, copy) SentryTraceContext *traceContext;
 
+@property (nullable, nonatomic, copy) NSDate *sentAt;
```

Comment:
`m`: We should add a comment explaining when to set this field. It must be as close as possible when actually sending the envelope and not when storing the envelope to disk. You could copy parts of the docs https://develop.sentry.dev/sdk/envelopes/.

Reply:
Took parts of the the docs which seemed appropriate here.